docs/guides/schemas.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I think this page should be merged with `docs/configuration/schema-configuration/`